### PR TITLE
Fix: unnecessary comma

### PR DESCRIPTION
**Pull request by Gootector**
Fix: unnecessary comma
https://www.microsoft.com/en-us/language/Search?&searchTerm=Please%20check%20your%20internet%20connection%20or%20try%20again%20later&langID=262&Source=true&productid=0

Greetings,
Gootector

Resolves: *(direct link to the issue)*

*(short description of the changes and the motivation to make the changes)*

- [x] I signed [CLA](https://musescore.org/en/cla)
- [x] I made sure the code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [x] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [x] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

File: src/project/internal/projectactionscontroller.cpp [hunk 1]
```diff
@@ -884,7 +884,7 @@ void ProjectActionsController::warnCloudIsNotAvailable()
 void ProjectActionsController::warnPublishIsNotAvailable()
 {
     std::string title = trc("project/save", "Unable to connect to MuseScore.com");
-    std::string msg = trc("project/save", "Please check your internet connection, or try again later.");
+    std::string msg = trc("project/save", "Please check your internet connection or try again later.");
```
Comment (Jojo-Schmitz) [inline src/project/internal/projectactionscontroller.cpp:887]: That string occurs twice, you fixed only one

Comment (Jojo-Schmitz) [reply src/project/internal/projectactionscontroller.cpp:887]: I take that back, that other scting os lacking the comma

**Comment by Gootector** [comment]
@cbjeukendrup

**Review by cbjeukendrup — APPROVED** [review]

**Comment by Gootector** [comment]
Thanks, @cbjeukendrup.

**Pull request merged**